csrankings-y.csv

```diff
@@ -8,6 +8,7 @@ Y. Raghu Reddy,IIIT Hyderabad,http://faculty.iiit.ac.in/~raghu.reddy,NOSCHOLARPA
 Y. Tsiatouhas,University of Ioannina,https://cs.uoi.gr/~tsiatouhas,gHsE8pYAAAAJ
 Y. Y. Yao,University of Regina,http://www.cs.uregina.ca/People/Profiles/YiyuYao.html,_aL0fcQAAAAJ
 Y.N. Srikant,IISc Bangalore,http://www.csa.iisc.ac.in/~srikant,NOSCHOLARPAGE
+Ya Zhang 0002,Shanghai Jiao Tong University,https://mediabrain.sjtu.edu.cn/yazhang,pbjw9sMAAAAJ

```

Comment:
In EE, not CS. No justification provided.

>  If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

This is the third failure to actually follow the directions. I am closing this PR, as with the previous ones.